src/array/kernel.cc

```diff
@@ -226,6 +226,23 @@ void ScatterAddDispatch(NDArray feat, NDArray idx, NDArray out) {
   });
 }
 
+/*! \brief Scatter Add (on first dimension) dispatch function on heterogeneous graph. */
+void ScatterAddDispatchHetero(HeteroGraphPtr graph,
```

Comment:
You can just write `UpdateGradMinMax_hetero` to keep it consistent with other places.

Reply:
Done